libs/api/Verifier.cpp

```diff
@@ -134,9 +134,7 @@ verify_byte_code(
 std::vector<uint8_t>
 convert_ebpf_program_to_bytes(const std::vector<ebpf_inst>& instructions)
 {
-    std::vector<uint8_t> retval(instructions.size() * sizeof(ebpf_inst));
-    memcpy(retval.data(), instructions.data(), retval.size());
-    return retval;
+    return {instructions.data(), instructions.data() + instructions.size() * sizeof(ebpf_inst)};
```

Comment:
Looks, good, but you will need to use something like: ``` reinterpret_cast<const uint8_t*>(instructions.data()) ``` to convert it from a const pointer to ebpf_inst to a const pointer to uint8_t.

Reply:
Thank you so much, I don’t have a compiler that meets the requirements for compiling this project, so I rely on the CI system to compile :)